packages/build/tests/install/snapshots/tests.mjs.md

```diff
@@ -639,7 +639,7 @@ Generated by [AVA](https://avajs.dev).
       Please make sure the file exists and its path is correctly spelled.␊
     ␊
       In file "packages/build/tests/install/fixtures/local_dep/functions/index.js"␊
-      Cannot find module './missing' from 'packages/build/tests/install/fixtures/local_dep/functions'␊
+      Cannot find module './missing.js' from 'packages/build/tests/install/fixtures/local_dep/functions'␊
```

Comment:
File extensions are required by `import()` (unlike `require()`), which explains this change.